package.json

```diff
@@ -35,7 +35,8 @@
       "temperature": "bricklets/Temperature.js",
       "digiin": "bricklets/IndDigital4In.js",
       "analogout": "bricklets/AnalogOut.js",
-      "indanalogout": "bricklets/IndAnalogOut.js"
+      "indanalogout": "bricklets/indAnalogOut.js",
```

Comment:
Ive tried in packages.json but node-red hangs when the case is correct. This was also the case before my first change, do you have an clue why ?